src/ray/gcs/tables.h

```diff
@@ -151,12 +150,12 @@ class ObjectTable : public Table<ObjectID, ObjectTableData> {
   /// objects from the object table. The callback will be called once per
   /// notification received on this channel.
   ///
-  /// @param subscribe_all
-  /// @param object_available_callback Callback to be called when new object
+  /// \param subscribe_all
+  /// \param object_available_callback Callback to be called when new object
   ///        becomes available.
-  /// @param done_callback Callback to be called when subscription is installed.
```

Comment:
Minor, but "becomes available" should be unindented (other places in this file as well)